package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "gym-master-back",
+  "version": "1.0.0",
+  "description": "Express server for gym master application",
+  "main": "server.js",
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development nodemon app.js",
+    "start": "cross-env NODE_ENV=production node app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Toujane/gym-master-back.git"
+  },
+  "keywords": [
+    "back",
+    "express",
+    "node",
+    "gym",
+    "master",
+    "toujane"
+  ],
+  "author": "Toujane (Blažek Ivan, Kunštek Antonio, Matijević Mislav)",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Toujane/gym-master-back/issues"
+  },
+  "homepage": "https://github.com/Toujane/gym-master-back#readme",
+  "dependencies": {
+    "bcrypt": "^5.0.1",
+    "bcryptjs": "^2.4.3",
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "cross-env": "^7.0.3",
+    "dotenv": "^16.0.0",
+    "express": "^4.17.3",
+    "express-async-errors": "^3.1.1",
+    "express-mongo-sanitize": "^2.2.0",
+    "express-rate-limit": "^6.3.0",
+    "helmet": "^5.0.2",
+    "jsonwebtoken": "^8.5.1",
+    "mongoose": "^6.3.1",
+    "morgan": "^1.10.0",
+    "nodemon": "^2.0.15",
```

Comment:
This should be a dev dependency!

Reply:
No idea why fricking GitHub mentions 4 lines. I specifically selected line 44: nodemon.

Reply:
It was a problem with depracated npm install. I moved packages to dev dependencies. :)